src/calibrate/readme.md

```diff
@@ -0,0 +1,51 @@
+# Calibrate microservice
+
+## To run in a virtual environment
+
+1. Add file `airqo-250220-5149c2aac8f2.json` to `jobs/`. Obtain from a team member or GCP.
+1. Create a virtual environment
+2. `pip install -r requirements/prod.txt` or `pip install -r requirements/dev.txt`
+3. `flask run`
+
+## To build and run with docker desktop
+
+(Uses "production" dockerfile.)
+
+1. Add file `airqo-250220-5149c2aac8f2.json` to `jobs/`. Obtain from a team member or GCP.
+1. `docker build -t calibrate .`
+2. `docker run -d -p 4001:4001 --env FLASK_APP=app.py --env FLASK_ENV=development --env MONGO_URI=mongodb://localhost:27017 calibrate`
+
+It is implicit that `mongodb` should be installed and running.
+
+## Check endpoint
+
+Make a "POST" request to http://localhost:4001/api/v1/calibrate with the following raw JSON payload in the body:
+
+```{json}
+{
+    "datetime": "2020-07-15 13:00:00",
+    "raw_values": [
+        {
+            "device_id": "aq_01",
+            "pm2.5": 44.12,
+            "pm10": 54.20,
+            "temperature": 25.3,
```

Comment:
This is great work! Just sharing some data engineering notes/thoughts about how this endpoint will be used within our data pipeline.

I can imagine that we have to first use the device ID or channel ID to search for the temperature and humidity readings for each device and then create a body to be used for this endpoint. If we use the channel ID, then we would also have to use the GET Devices endpoint to search for device name of each channel ID.

A few more questions/thoughts...

1. One thing to note is that there might be a good number of devices without temperature and humidity readings, what do we do for them?
2. Shall we be using the recent transform endpoint to read the temperature and humidity readings?
3. In this README, could we also showcase a sample of the response message after making the POST?

Thanks.

Reply:
@Baalmart - I'm documenting how it is at the moment. These are good points. Perhaps they could become a Jira issue for future resolution as some effort would be involved to implement.

Reply:
@Priscilla-AL - can you shed any light on this? I did not intend to change the behaviour of the microservice (only to reduce image size by having fewer requirements), but we have no version on staging to compare with so we're not sure if the output is correct in all circumstances.

Reply:
I have raised https://airqoteam.atlassian.net/browse/PLAT-526 to address this later, seperately from the change to requirements. If this approach is OK, I suggest the PR can be approved.